xrpl/asyncio/ledger/main.py

```diff
@@ -74,18 +86,66 @@ async def get_fee(
     if not response.is_successful():
         raise XRPLRequestFailureException(response.result)
 
-    result = response.result["drops"]
-    if fee_type == "open":
-        fee = cast(str, result["open_ledger_fee"])
-    elif fee_type == "minimum":
-        fee = cast(str, result["minimum_fee"])
-    else:
-        raise XRPLException(
-            f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
-            "valid option."
+    result = response.result
+    drops = result["drops"]
+    if fee_type:
+        if fee_type == "open":
+            fee = cast(str, drops["open_ledger_fee"])
+        elif fee_type == "minimum":
+            fee = cast(str, drops["minimum_fee"])
+        else:
+            raise XRPLException(
+                f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
+                "valid option."
+            )
+    else:  # use https://gist.github.com/WietseWind/3e9f9339f37a5881978a9661f49b0e52
```

Comment:
This credit was a little confusing, maybe having it on a new line with a bit more context like:
```
# Code originally written here: https://gist.github.com/WietseWind/3e9f9339f37a5881978a9661f49b0e52
```